chromiumoxide_fetcher/src/browser/runtime.rs

```diff
@@ -0,0 +1,103 @@
+use std::path::{Path, PathBuf};
+
+use anyhow::Context;
+
+use super::ZipArchive;
+
+pub struct BrowserFetcherRuntime;
```

Comment:
We might want an async trait for that if we want to enable both runtimes at the same time?